### PR TITLE
Create contracts folder when SpringCloudContract Verifier is selected

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

**Comment by wilkinsona** [comment]
Thank you, @eddumelendez. This seems like a good idea to me, but I am far from being a SCC expert. WDYT, marcingrzejszczak?

File: start-site/src/test/java/io/spring/start/site/extension/SpringCloudContractProjectContributorTests.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+/**
+ * Tests for {@link SpringCloudContractProjectContributor}.
+ *
+ * @author Eddú Meléndez
+ */
+class SpringCloudContractProjectContributorTests extends AbstractExtensionTests {
+
+	@Test
+	void contractsDirectoryIsCreatedWithSpringCloudContractVerifier() {
```
Comment (marcingrzejszczak) [inline start-site/src/test/java/io/spring/start/site/extension/SpringCloudContractProjectContributorTests.java]: I don't know what this `web` component means, but you can have contracts for a messaging application that doesn't have a web interface.

Comment (eddumelendez) [reply start-site/src/test/java/io/spring/start/site/extension/SpringCloudContractProjectContributorTests.java]: this test makes sure that when `cloud-contract-verifier` dependency is added `contracts` folder is created. it doesn't matter if it is a web application or not.

Comment (wilkinsona) [reply start-site/src/test/java/io/spring/start/site/extension/SpringCloudContractProjectContributorTests.java]: To clarify things, `"web"` could be removed and the test will still be valid.

File: start-site/src/test/java/io/spring/start/site/extension/SpringCloudContractProjectContributorTests.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		ProjectRequest request = createProjectRequest("web", "cloud-contract-verifier");
+		ProjectStructure structure = generateProject(request);
+		assertThat(structure.getProjectDirectory().resolve("src/test/resources/contracts")).exists().isDirectory();
+	}
+
+	@Test
+	void contractsMigrationDirectoryIsNotCreatedIfSpringCloudContractVerifierIsNotRequested() {
```
Comment (wilkinsona) [inline start-site/src/test/java/io/spring/start/site/extension/SpringCloudContractProjectContributorTests.java]: No need for `Migration` here?

Comment (eddumelendez) [reply start-site/src/test/java/io/spring/start/site/extension/SpringCloudContractProjectContributorTests.java]: that's right. fixed

**Comment by eddumelendez** [comment]
just realized my branch is named `sprincloudcontract_contributor` instead of `springcloudcontract_contributor` :c

**Comment by snicoll** [comment]
Thanks again @eddumelendez

**Pull request closed without merging**